css/_fonts.scss

```diff
@@ -0,0 +1,24 @@
+@font-face {
+  font-family: 'TTNorms';
```

Comment:
Please, add space. It should be 'TT Norms'.
Fix all the typos.

Reply:
Thank you. All done.